node-test/src/test/scala/org/bitcoins/node/networking/ClientTest.scala

```diff
@@ -100,8 +101,9 @@ class ClientTest
   }
 
   override def afterAll: Unit = {
-    bitcoindRpcF.flatMap(_.stop())
-    bitcoindRpc2F.flatMap(_.stop())
+    implicit val ec = system.dispatcher
+    Await.ready(bitcoindRpcF.flatMap(ChainUnitTest.destroyBitcoind(_)(system)), 10.seconds)
```

Comment:
There's a method `BitcoindRpcTestUtil.stopServers` that can be used for this purpose. It adds some logging of what happens, and also deletes the temporary data directory we've created. Do you mind using that method instead?